doc/classes/OS.xml

```diff
@@ -1162,6 +1174,21 @@
 		<constant name="MONTH_DECEMBER" value="12" enum="Month">
 			December.
 		</constant>
+		<constant name="APPLICATION_HANDLE" value="0" enum="HandleType">
+			Application handle: HINSTANCE, NSApplication*, UIApplication*, JNIEnv* ...
```

Comment:
All these should use `[code]` markup. Might need some more explanations about what each handle is and can be used for. (And then yeah a list of possible handle types depending on the platform.)

Reply:
The docs should likely also list exactly what is returned on each platform, instead of potential handles that might be returned in the future, so that users know what they'll get without having to read the Godot source code.